service/surf/apps/locale/views.py

```diff
@@ -21,5 +21,6 @@ def get_localisation_strings(request, locale):
     # Here we inject the privacy statement as a translation
     # in order for the frontend to simply switch between translations
     privacy_statement = PrivacyStatement.objects.get_latest_active()
-    data["html-privacy-info"] = getattr(privacy_statement, locale)
+    if privacy_statement:
```

Comment:
Niet zo zeker van deze. Als er geen privacy statement is, dan moeten we toch op zijn minst een signaal naar Sentry sturen? We mogen helemaal niet opereren zonder privacy statement namelijk. 

Het werkt voor andere tests met locales. Wat is er precies veranderd?

We zouden wel de huidige privacy statement en goals kunnen dumpen en inladen via invoke srv.setup. Zoiets doen we ook met hrv.setup. Het zijn vertalingen die altijd in de database blijven staan, omdat gebruikers "ja" zeggen tegen een specifieke (vertaalde) versie. Dus een relatie met User model lijkt me onvermijdelijk.

Reply:
De vertalingen werkten niet in e2e tests, omdat er geen privacy statements in de database zitten en dan gaat het locale endpoint keihard kapot. Als er altijd een privacy statement moet zijn dan moet ik het overal maar gaan seeden. Er bestaat wel al een fixture voor namelijk. Ik zal het aanpassen.